### PR TITLE
QNTM 4211 - File Path node is missing in Player

### PR DESCRIPTION
**Pull request by saintentropy**
### Purpose

The purpose of this PR is to allow the File Path and Directory Path nodes to be set as "Is Input" in the UI so that they can be used with Dynamo Player.  This was supported in Dynamo 1.3.  

### Declarations

Check these if you believe they are true

- [X] The code base is in a better state after this PR
- [X] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [X] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions), and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@ramramps @QilongTang 

### FYIs

@jnealb @Racel

### CODE REVIEW

File: src/Libraries/CoreNodeModels/Input/FileSystem.cs [hunk 1]
```diff
@@ -111,15 +117,6 @@ public Filename() : base("Filename")
         {
             ShouldDisplayPreviewCore = false;
         }
-
-        /// <summary>
-        ///     Indicates whether node is input node.
-        ///     Used to bind visibility of UI for user selection.
-        /// </summary>
-        public override bool IsInputNode
```
Comment (ramramps) [inline src/Libraries/CoreNodeModels/Input/FileSystem.cs]: this makes sense to me

File: src/Libraries/CoreNodeModels/Input/FileSystem.cs [hunk 2]
```diff
@@ -88,6 +88,12 @@ protected override void DeserializeCore(XmlElement nodeElement, SaveContext cont
                 HintPath = subNode.InnerText;
             }
         }
+
+        public override NodeInputData InputData
+        {
+            //There is not yet an appropriate input type for CoGs related to File Path or Directory Path
+            get { return null; }
```
Comment (ramramps) [inline src/Libraries/CoreNodeModels/Input/FileSystem.cs]: well, is this required for hotfix?

Comment (mjkkirschner) [reply src/Libraries/CoreNodeModels/Input/FileSystem.cs]: really? aren't they just strings?

Comment (saintentropy) [reply src/Libraries/CoreNodeModels/Input/FileSystem.cs]: I am fine leaving this... I just seemed to not make sense to me to have these nodes as inputs in the CoGs context.  Easy to remove this override if you all want me to

Comment (ramramps) [reply src/Libraries/CoreNodeModels/Input/FileSystem.cs]: if this is not related to this hotfix, can your remove it?

Comment (saintentropy) [reply src/Libraries/CoreNodeModels/Input/FileSystem.cs]: Yep

**Comment by ramramps** [comment]
LGTM. Please cherry pick this in RC2.0.1 branch.
@saintentropy

**Comment by QilongTang** [comment]
LGTM

**Comment by saintentropy** [comment]
@ramramps @QilongTang will do as soon as tests pass

**Pull request merged**